imap_processing/swapi/swapi_cdf_attrs.py

```diff
@@ -0,0 +1,106 @@
+"""Shared attribute values for SWAPI CDF files."""
+
+from imap_processing.cdf.defaults import GlobalConstants
+from imap_processing.cdf.global_attrs import (
+    AttrBase,
+    GlobalDataLevelAttrs,
+    GlobalInstrumentAttrs,
+    ScienceAttrs,
+)
+from imap_processing.swapi import __version__
+
+text = (
+    "The Solar Wind and Pickup Ion (SWAPI) instrument "
+    "measures several different elements of the solar "
+    "wind, including hydrogen (H) and helium (He) ions,"
+    " and, on occasion, heavy ions produced by large "
+    "events from the Sun. SWAPI also measures "
+    "interstellar pickup ions (PUIs), whose material "
+    "comes from beyond our solar system and moves along"
+    " with the solar wind. Its data will provide "
+    "information about the local ion conditions, such "
+    "as temperature, density, and speed. This will also"
+    " be used in the I-ALiRT data stream allowing space"
+    " weather to be measured in real-time. The data from"
+    " SWAPI will be valuable for understanding how the "
+    "solar wind changes in response to the Sun's behavior"
+    " over time. SWAPI's first-ever high time resolution "
+    "measurements of helium PUIs will provide new insights"
+    " into physical processes that accelerate charged "
+    "particles and shape and change our global heliosphere."
+    "See https://imap.princeton.edu/instruments/swapi for "
+    "more details."
+)
+
+swapi_base = GlobalInstrumentAttrs(
+    __version__,
+    "SWAPI>The Solar Wind and Pickup Ion",
+    text,
+    "Particles (space)",
+)
+
+# dimensions attributes
+energy_dim_attrs = AttrBase(
+    validmin=0,
+    validmax=72,
+    format="I2",
+    var_type="support_data",
+    fieldname="Energy Step",
+    catdesc="Energy step number.",
```

Comment:
Probably should be a bit more detailed, like "Energy steps at 72 energies from {x}-{y}eV", whatever the min and max energy are in the bins

Reply:
Good point. I don't know the exact energy level they use yet. These are id that will map to energy value in their lookup table. What you do you think if I change descriptions like this
```suggestion
    catdesc="Energy step id for lookup table",
```